src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -290,9 +291,6 @@ public void deleteRoles(final Node node) throws RepositoryException {
         } catch (final RepositoryException ex) {
             LOGGER.debug("Exception finding effective ACL: {}", ex);
```

Comment:
Please change to:
`"Exception finding effective ACL: {}", ex.getMessage()`
Dumping stacktraces into the log file convolutes debugging.